go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/envoyproxy/xds-relay
+
+go 1.12
```

Comment:
why so old? Does this have to go in lockstep with `go-control-plane`?

Reply:
No there shouldn't be any reason it can't be more recent. This is just what it defaulted to when I ran `go mod init`. Could be my version of Go, I'll upgrade this to 1.14.